gc3libs/backends/batch.py

```diff
@@ -402,9 +449,8 @@ def submit_job(self, app):
         # Create the remote directory.
         try:
             self.transport.connect()
-
-            cmd = "mkdir -p $HOME/.gc3pie_jobs;" \
-                " mktemp -d $HOME/.gc3pie_jobs/lrms_job.XXXXXXXXXX"
+            cmd = "mkdir -p {0}/.gc3pie_jobs;" \
+                " mktemp -d {0}/.gc3pie_jobs/lrms_job.XXXXXXXXXX".format(self.spooldir)
```

Comment:
Nope.

We need the working directory on a batch system to be shared across frontend and compute nodes.  `/home` and such typically are -- `/var/tmp` and `$TMPDIR` typically are *not*.

So we should either have a different default for "spooldir" on batch systems, or have a different default altogether (e.g., `$HOME/` instead of `$TMPDIR`)

Reply:
I see, but then we just need to change the default vaue in the Default class in gc3libs/__init__.py, no effect on batch.py

Reply:
Yes, but the question is: is there a default that is good for *all* cases?

Reply:
quick answer: no
but then I would use $home as default - not even try using $tmpdir